MyFirstSlideshow/Base.lproj/Main.storyboard

```diff
@@ -1,26 +1,158 @@
+                                <rect key="frame" x="80.5" y="490" width="79" height="10"/>
+                                <color key="backgroundColor" red="0.2307143807" green="0.76754516360000002" blue="0.5560275912" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                            </view>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="8cG-cP-Ogc">
+                                <rect key="frame" x="160.5" y="490" width="79.5" height="10"/>
+                                <color key="backgroundColor" red="0.2307143807" green="0.76754516360000002" blue="0.5560275912" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                            </view>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="YEe-M7-DE9">
+                                <rect key="frame" x="241" y="490" width="79" height="10"/>
+                                <color key="backgroundColor" red="0.2307143807" green="0.76754516360000002" blue="0.5560275912" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                            </view>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="UHf-17-dDN">
+                                <rect key="frame" x="0.0" y="522" width="320" height="30"/>
+                                <state key="normal" title="Download !"/>
+                                <connections>
+                                    <action selector="onButtonTwoClicked:" destination="BYZ-38-t0r" eventType="touchUpInside" id="7Md-BS-A5b"/>
+                                </connections>
+                            </button>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="uiA-dL-zpI">
+                                <rect key="frame" x="0.0" y="252" width="320" height="30"/>
+                                <state key="normal" title="Download !"/>
+                                <connections>
+                                    <action selector="onButtonOneClicked:" destination="BYZ-38-t0r" eventType="touchUpInside" id="MzH-rB-QVV"/>
+                                </connections>
+                            </button>
+                        </subviews>
                         <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                        <constraints>
+                            <constraint firstItem="TWd-zk-mdW" firstAttribute="width" secondItem="OSy-Ka-ZuK" secondAttribute="width" id="18N-Qb-7ke"/>
```

Comment:
its always better not to define width ,because width and height changes according to screen size